spec/rails_helper.rb

```diff
@@ -535,7 +552,7 @@ class SpecTimeoutError < StandardError
 
     driver = [:selenium]
     driver << :mobile if example.metadata[:mobile]
-    driver << :chrome
+    driver << (aarch64? ? :firefox : :chrome)
```

Comment:
My main worry is that we're going to end up with people opening topics on Meta like "specs are failing on Firefox, plz fix".

Can we print a warning here to make it clear that we do not support Firefox as a driver, and that test failures are expected?

```suggestion
    driver << (aarch64? ? :firefox : :chrome)
    if driver == :firefox
      STDERR.puts "WARNING: Running system specs is not officially supported. Some tests will fail."
    end
```

Reply:
Ah yes, good thinking :+1: :grin:

Reply:
Done :)